packages/ui/src/components/menu-item/menuItem.vue

```diff
@@ -193,8 +192,10 @@
       margin-left: auto;
     }
 
-    &:deep(.acv-icon) {
+    .acv-icon {
```

Comment:
can we not set styles on alien component here? create internal class `icon` for example and set styles on it